storybook/src/components/form/form-group-input-number.css-template.ts

```diff
@@ -6,22 +6,43 @@ import { ComponentImplementation } from "../../templates";
 export const cssFormGroupInputNumber: ComponentImplementation<FormGroupInputNumber<TemplateResult>> = {
   component: "formGroupInputNumber",
   implementation: "html-css",
-  template: ({ inputNumberTemplate }) =>
+  template: ({ buttonTemplate }) =>
     function formGroupInputNumberTemplate(formGroup) {
       return html`
         <div class="form-group dso-input-number">
           <div class="dso-label-container">
             <label for=${formGroup.id} class="control-label"> ${formGroup.label} </label>
           </div>
           <div class="dso-field-container">
-            ${inputNumberTemplate({
-              id: formGroup.id,
-              min: formGroup.min,
-              max: formGroup.max,
-              count: formGroup.count,
-              minusButtonInactive: formGroup.minusButtonInactive,
-              plusButtonInactive: formGroup.plusButtonInactive,
-            })}
+            <div class="dso-input-number">
```

Comment:
Input Number wordt gefuseerd met List Button, niet met Form Group: Input Number.

De Form Group componenten zijn wrappers om componenten in het formulier te gebruiken. Als Input Number niet bestaat, is Form Group: Input Number een overbodig component geworden. In die geest hadden we volgens mij ook bedacht om dit component ook te verwijderen

Reply:
Form Group: Input Number is verwijderd